other/gauss_easter.py

```diff
@@ -0,0 +1,48 @@
+"""
+https://en.wikipedia.org/wiki/Computus#Gauss'_Easter_algorithm
+"""
+import math
+from datetime import datetime, timedelta
+
+
+def gauss_easter(year: int) -> datetime:
+    """
+    Calculation Gregorian easter date for given year
+
+    >>> gauss_easter(2007)
+    datetime.datetime(2007, 4, 8, 0, 0)
+
+    >>> gauss_easter(2008)
+    datetime.datetime(2008, 3, 23, 0, 0)
+
+    >>> gauss_easter(2020)
+    datetime.datetime(2020, 4, 12, 0, 0)
+
+    >>> gauss_easter(2021)
+    datetime.datetime(2021, 4, 4, 0, 0)
+    """
+    a = year % 19
+    b = year % 4
+    c = year % 7
+    k = math.floor(year / 100)
+    p = math.floor((13 + 8 * k) / 25)
+    q = k / 4
+    M = (15 - p + k - q) % 30
+    N = (4 + k - q) % 7
+    d = (19 * a + M) % 30
+    e = (2 * b + 4 * c + 6 * d + N) % 7
```

Comment:
Single letter variable names are old school and they force the reader to guess a lot to make sense of this algorithm.

Reply:
Should be better now.